src/framework/lib/genesisframework/utils.rb

```diff
@@ -34,11 +34,15 @@ def self.collins
       end
 
       def self.facter
-        if @@facter.nil?
-          @@facter = Facter.to_hash
-        end
+        if config_cache[:facter_cache] == false
```

Comment:
If we do what you suggest, then the default behavior will be to disable the cache (since if it is not set, your conditional will evaluate to true).  I think what @defect has is better since it'll require that the cache is explicitly disabled.